WcaOnRails/config/storage.yml

```diff
@@ -0,0 +1,10 @@
+local:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+server:
```

Comment:
I think it would be nice to call this `amazon:` instead of `server:`. That fits with the suggestion I see here: https://edgeguides.rubyonrails.org/active_storage_overview.html#amazon-s3-service

Reply:
Sure, that sounds good!